Casks/confluent-hub-client.rb

```diff
@@ -1,6 +1,6 @@
 cask 'confluent-hub-client' do
-  version '5.0.1'
-  sha256 'a640fbbdf6c750c82716a29afc2b76847f622218dd8521cdd9dccaf509ebc13d'
+  version '5.3.0-rc2'
```

Comment:
don't think we should be publishing an rc2, i assume both lines may need to be updated to match the final 5.3 release?

Reply:
@ewencp you are right, we can re-run https://jenkins.confluent.io/job/hub-client-prepare-release-brew/ once we have finalized the 5.3.

Reply:
see confluentinc/homebrew-confluent-hub-client#6